src/OpenSage.Game/Data/Ini/CommandButton.cs

```diff
@@ -622,5 +697,192 @@ public enum CommandButtonRadiusCursorType
 
         [IniEnum("DOMINATE"), AddedIn(SageGame.Bfme)]
         Dominate,
+
+        //should this realy be enum values? or just names/strings
+        [IniEnum("ScatterAreaRadiusCursor"), AddedIn(SageGame.Bfme2)]
+        ScatterAreaRadiusCursor,
+
```

Comment:
Not sure if this Enum should be just a string value

Reply:
Pretty sure it was an enum, which has caused modders a lot of problems because it can't be extended. Let's keep it as an enum, but hopefully we can lift that restriction later.

Reply:
ok thx